packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SentenceScopeHandler/SentenceSegmenter.ts

```diff
@@ -0,0 +1,60 @@
+import * as sbd from "sbd";
+import { MatchedText } from "../../../../util/regex";
+
+// A sentence starts with a letter
+const leadingOffsetRegex = /\p{L}/u;
+// A line with no letters is invalid and breaks sentences
+const invalidLineRegex = /(\n[^\p{L}]*\n)/gu;
+
+const options: sbd.Options = {
+  ["newline_boundaries"]: false,
+  ["preserve_whitespace"]: true,
+};
+
+export default class SentenceSegmenter {
+  *segment(text: string): Iterable<MatchedText> {
+    const sentences = sbd.sentences(text, options);
```

Comment:
Is there any way we can fall back to intl segmenter if the language isn't English?  Not sure we want to roll a language detector into our code as well 🤔

Reply:
I don't think we should do that. Many languages including Swedish has very similar structure to english. intl segmenter was truly atrocious in many aspects and would require special post processing before I think it would be useful.

Reply:
Ok let's leave it for now.  If people file issues for supporting other languages, we can look for better segmenters for those languages and deal with language detection then.

Reply:
Sounds good